scripts/plugin_loader.py

```diff
@@ -64,6 +63,19 @@ def verify_directories_exist(dirs):
             raise ValueError("{} does not exist".format(path))
 
 
+def get_plugin_folder_paths(**kwargs):
```

Comment:
It's not clear that `**kwargs` is necessary here, as it seems to expect only a single argument.  If you're trying to have an optional arg that accepts a default, writing it this way is more conventional:

```python
def get_plugin_folder_paths(base_path=BASE_DIR):
   region_json = to_json(REGION_FILE)   
   plugin_directories = get_plugin_directories(region_json, base_path)
   ...
```

Reply:
A ha! Thanks, that is what I was trying to go for.